jest.config.js

```diff
@@ -6,10 +6,10 @@ module.exports = {
     "^.+\\.(ts|tsx)?$": "ts-jest",
   },
   collectCoverageFrom: [
-    "src/**/*.ts",
+    "**/src/**/*.ts",
```

Comment:
Does this work when running Jest for a single package as well?

Reply:
Yep - seems to work fine (i.e. running `npm run test-unit` from within the `/packages/browser` package).